webapps-schema/src/main/java/io/camunda/webapps/schema/entities/tasklist/TasklistEntity.java

```diff
@@ -29,42 +30,45 @@ public abstract class TasklistEntity<T extends TasklistEntity<T>> extends Abstra
   public TasklistEntity() {}
 
   public TasklistEntity(
-      final String id, final long key, final String tenantId, final int partitionId) {
+      final String id, final Long key, final String tenantId, final int partitionId) {
     this.id = id;
     this.key = key;
     this.tenantId = tenantId;
     this.partitionId = partitionId;
   }
 
-  public TasklistEntity(final String id, final long key, final String tenantId) {
+  public TasklistEntity(final String id, final Long key, final String tenantId) {
```

Comment:
🤔 Shouldn't we delete this entity? Or do we keep the entity but not having the respective index?

Reply:
Ah can we maybe merge it with the TaskEntity?

Reply:
`TasklistEntity` serves as a base class for all entities related to previous Tasklist indices, it contains the three parameters that are common to all entities (id, key and partitionId). We could create a base class for both Operate and Tasklist entities and use as a baseline in later iterations.